lld/ELF/Config.h

```diff
@@ -353,22 +353,8 @@ struct Configuration {
   // Size of a CHERI capability
   int capabilitySize = 0;
 
-  inline bool isCheriABI() const { return cheriABI; }
-  // We need to set the searchPaths before createFiles() is called since linker
-  // scripts might contain INPUT() commands. Add a getter and setter for
-  // CheriABI to ensure this is always the case
-  inline void setIsCheriABI(bool set) {
```

Comment:
I think the problem might have been linker scripts since they used to ignore -L flags. Since this was fixed upstream a while ago it should be safe to remove this ugly hack.